platform-validation/scripts/aziot-compatibility.sh

```diff
@@ -553,6 +553,42 @@ check_shared_library_dependency_display_util() {
     esac
 }
 
+check_package_manager() {
+    not_found=0
+    result="$(need_cmd apt-get)"
+    if [ $? != 0 ] ; then
+        wrap_debug "apt-get package manager is not present."
+        result="$(need_cmd dpkg)"
+        if [ $? != 0 ]; then
+            wrap_debug "dpkg package manager is not present."
+            result="$(need_cmd yum)"
+            if [ $? != 0 ]; then
+                wrap_debug "yum package manager is not present."
+                result="$(need_cmd rpm)"
+                if [ $? != 0 ]; then
+                    wrap_debug "rpm package manager is not present."
+                    not_found=1
+                else
+                    wrap_debug "rpm package manager is present."
+                fi
+            else
+                wrap_debug "yum package manager is present."
+            fi
+        else
+            wrap_debug "dpkg package manager is present"
+        fi
+    else
+        wrap_debug "apt-get package manager is present"   
+    fi
+
+    if [ "$not_found" -eq 1 ]; then
+        wrap_warning "Install a package manager for your distribution."
```

Comment:
I think we should say something like 
*IoT Edge supports the following package types [*deb, *rpm] and following package managers [*apt-get], We have identified that this device does not have support for the supported package type, Please head to aka.ms/iotedge for instructions on how to build the iotedge binaries from source*